environments/dev-env/stages/edge-placement-s.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: edge.kcp.io/v1alpha1
+kind: EdgePlacement
+metadata:
+  name: edge-placement-s
+spec:
+  locationSelectors:
+  - matchLabels: {"env":"prod","extended":"si"}
```

Comment:
This `matchLabels` doesn't select `location-g`, which is differrent with [our designed scenario](https://github.com/kcp-dev/edge-mc/blob/9a17ccbec614156a58f24efbd436e37ff9b5d115/docs/content/en/docs/Coding%20Milestones/PoC2023q1/example1.md?plain=1#L473).

Reply:
You're correct! Just fixed it